gravitee-apim-portal-webui-next/src/entities/log/messageLog.ts

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (C) 2025 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { LogsResponseMetadata } from './log';
+
+export interface AggregatedMessageLog {
+  requestId: string;
+  timestamp: string;
+  clientIdentifier: string;
+  correlationId: string;
+  parentCorrelationId: string;
+  operation: MessageOperation;
+  entrypoint: Message;
+  endpoint: Message;
+}
+
+export interface Message {
+  connectorId: string;
+  timestamp: string;
+  id: string;
+  payload: string;
+  isError: boolean;
+  headers: Record<string, string[]>;
+  metadata: Record<string, string>;
+}
+
+export type ConnectorType = 'ENDPOINT' | 'ENTRYPOINT';
+export type MessageOperation = 'SUBSCRIBE' | 'PUBLISH';
+
+export interface AggregatedMessageLogsResponse {
+  data: AggregatedMessageLog[];
+  links?: unknown;
```

Comment:
is it really unknown, can we have something more specific ? 🤔 

Maybe we should create the interface declared in the portal-openapi.yaml for the links to be able to reuse it in this kind of case for example. wdyt ?